Hummingbird.docc/Articles/Recommendations.md

```diff
@@ -0,0 +1,342 @@
+## Dependencies
+
+Hummingbird won't enforce any dependencies, or provide special treatment for dependencies on a framework level. However, we'll provide some guidelines and tips for _common_ dependencies and how to use them.
+
+## Project Structure
+
+The project structure is a recommended project layout, defining what modules are recommended to create and how these modules are structured in terms of folders and files.
+
+### Module Structure
+
+Every Hummingbird projects neds a primary executable target. This is the **MyApp** target in the example below.
+
+If a user has an OpenAPI specification for their API, a separate OpenAPI module is recommended based on [swift-openapi-generator](https://github.com/apple/swift-openapi-generator). This module will be used to generate the API server code.
+
+```
+├── Sources
+│   ├── MyApp
+│   └── MyAppOpenAPI # Optional
```

Comment:
Template create a folder `MyAppAPI`. Either we change it here or change it in the template